producers/src/unit-test/java/uk/gov/moj/sdt/producers/sdtws/SdtInternalEndpointPortTypeTest.java

```diff
@@ -69,23 +83,50 @@ class SdtInternalEndpointPortTypeTest extends AbstractSdtUnitTestBase {
     @Mock
     private IWsUpdateItemHandler mockUpdateItemHandler;
 
+    @Mock
+    private Logger mockLogger;
+
+
     /**
      * Set up common for all tests.
      */
     @BeforeEach
     @Override
     public void setUp() {
-        portType = new SdtInternalEndpointPortType(mockUpdateItemHandler);
+        try (MockedStatic<LoggerFactory> mockLoggerFactory = Mockito.mockStatic(LoggerFactory.class)) {
+            mockLoggerFactory.when(() -> LoggerFactory.getLogger(any(Class.class)))
+                .thenReturn(mockLogger);
+            when(mockLogger.isDebugEnabled())
+                .thenReturn(true);
```

Comment:
This appears to have been added to ensure that the isDebugEnabled() branches are covered, but none of the tests carry out any checks to confirm this. Instead of this you might want to consider using the ListAppender method described at https://stackoverflow.com/questions/1827677/how-to-do-a-junit-assert-on-a-message-in-a-logger to check the contents of the log. See TargetApplicationSubmissionServiceTest.processRequestToSubmitAllSuccess() for an example.

Reply:
Mockito verify() statements have been added to the tests to confirm that the expected branches have been executed.